src/components/homepage/PinnedTicket.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+  <div class="notification is-info">
+    <div class="columns is-mobile">
+      <div class="column is-10">
+        <p class="is-size-5">Ticket {{ number }} | {{ tags }} | {{ title }}</p>
+        <!-- TODO: Add due dates to tickets on backend -->
+        <!-- <p>
+          DUE DATE
+        </p> -->
+        <!-- <p>Due May 25, 2020.</p> -->
+      </div>
+      <div class="column is-2">
+        <span @click="deletePin"><i class="bx bxs-pin bx-sm"></i></span>
+      </div>
+      <!-- TODO: Determine how to expose ticket completion
+                     endpoints on the homepage -->
+      <!-- <div class="column">
+                        <div class="buttons">
+                           <div class="button is-static">
+                              <p>Complete</p>
+                           </div>
+                        </div>
+                     </div> -->
+      <!-- TODO: Progress indicator -->
+      <!-- <div class="columns">
+                     <div class="column">
+                        <p class="is-size-6 has-text-weight-semibold">50% Complete (2/4)</p>
+                        <progress class="progress" value="50" max="100">50%</progress>
+                     </div>
+                  </div> -->
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { computed, defineComponent, PropType } from "vue";
+
+import store from "@/store";
+import { deletePinnedTicket, PinnedTicketRecord } from "@/api/pinned";
+
+const pinnedTicketComponent = defineComponent({
+  name: "PinnedTicket",
+  props: {
+    pinRecord: {
+      type: Object as PropType<PinnedTicketRecord>,
+      required: true
+    }
+  },
+  emits: ["pinnedDidUpdate"],
+  setup(props, context) {
+    const number = computed(() => props.pinRecord.ticket.ticket_number);
+    const tags = computed(() => {
```

Comment:
I think a v-for would be a lot cleaner here.

Reply:
This is used within a v-for on the homepage, not sure what you mean by this.

Reply:
I'm saying that I would have used a v-for for the tags instead of building a string fro them.

Reply:
I don't understand why you're making computed properties when you could access the props directly from the template. Is this a best practice that I don't understand?

Reply:
Ah, that's a good point, I'll implement that before merging.

Reply:
And no, that was me forgetting props were directly accessible :P will fix